src/screens/WalletScreen.js

```diff
@@ -71,6 +73,15 @@ const WalletScreen = ({
         </HeaderButton>
       </Header>
       <AssetList
+        fetchData={() => {
+          // note: every time this is called it sets a new interval. this is a memory leak.
+          accountUpdateAccountAddress(accountInfo.address, 'BALANCEWALLET');
+          return new Promise((resolve) => {
```

Comment:
this promise + timeout is here just to demonstrate the refresh indicator. `accountUpdateAccountAddress` does not return a promise, so the refresh indicator does not stay visible while the account is being updated.